src/main/java/domain/Ladder.java

```diff
@@ -1,28 +1,39 @@
 package domain;
 
 import domain.line.RowLine;
-import domain.line.RowLineGenerator;
+import domain.line.RowLinesGenerator;
+import domain.name.Names;
+import domain.prize.Prizes;
 
-import java.util.ArrayList;
 import java.util.List;
 
 public class Ladder {
 
+    private static final int WIDTH_IDX = 1;
```

Comment:
```suggestion
    private static final int WIDTH_INDEX = 1;
```

축약어 금지입니다. 내가 아는 축약어를 모든 사람이 다 알고 있을거라 가정하면 안 됩니다.